source/django_views.rst

```diff
@@ -350,76 +320,75 @@ Then play with voting on them and invite others to do the same.
 
    .. code-block:: python
 
+        from django.http import Http404
         from django.http import HttpResponseRedirect
-        from django.core.urlresolvers import reverse
+
         from django.shortcuts import get_object_or_404
+        from django.shortcuts import render
+        from django.urls import reverse
 
         from polls.models import Choice
-        from django.http import Http404
-        from django.shortcuts import render
         from polls.models import Poll
 
-
         def index(request):
             latest_poll_list = Poll.objects.all().order_by('-pub_date')[:5]
             return render(
                 request,
                 'polls/index.html',
                 {'latest_poll_list': latest_poll_list})
 
-
         def detail(request, poll_id):
             try:
                 p = Poll.objects.get(id=poll_id)
             except Poll.DoesNotExist:
                 raise Http404
             return render(request, 'polls/detail.html', {'poll': p})
 
-
         def results(request, poll_id):
             p = get_object_or_404(Poll, id=poll_id)
             return render(request, 'polls/results.html', {'poll': p})
 
-
         def vote(request, poll_id):
             p = get_object_or_404(Poll, id=poll_id)
             try:
                 selected_choice = p.choice_set.get(id=request.POST['choice'])
             except (KeyError, Choice.DoesNotExist):
-                # if user chooses a wrong option, show error
+                # If the user chooses the wrong option, show error
```

Comment:
Sometimes `the user` is used and sometimes `a user`.
I think `user` is good enough, because in technical documents `a` and `the` is often omitted, but
it is worth checking with a native speaker.

Reply:
Change here is just a result of copy/paste [here](https://github.com/ggcarrots/django-carrots/blob/1fc5c81d146454674b3fe2b2d756527b99a7bc4a/source/django_views.rst#form-management) (_Correct the function `vote` as below_ -> full _`polls/views.py`_) so correction should be made here and there ;)